src/mpyl/steps/deploy/dagster.py

```diff
@@ -204,22 +204,8 @@ def execute(self, step_input: Input) -> Output:
         """
         results = []
         properties = step_input.run_properties
-        context = get_cluster_config_for_project(
```

Comment:
Feel free to delete these support functions as soon as they're no longer used (hard to check for me)

Reply:
Still in use